lib/pod.php

```diff
@@ -137,7 +140,14 @@ function is_main_root () {
     }
 
     function get_subsites () {
-        return [];     # TODO: repair
+      $retvals = array();
+      $currentUrl = 'https://' . $_SERVER['HTTP_HOST'] . '.epfl.ch' . substr($_SERVER['REQUEST_URI'], 0, strpos($_SERVER['REQUEST_URI'],"wp-admin"));
```

Comment:
Just wondering if we can get something dynamic here, so it might work with fsd.team or wp-httpd.